staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types_jsonschema.go

```diff
@@ -196,6 +196,13 @@ type ValidationRule struct {
 	// If unset, the message is "failed rule: {Rule}".
 	// e.g. "must be a URL with the host matching spec.host"
 	Message string
+	// MessageExpression represents a CEL expression that can be evaluated when validation fails, whereas Message
+	// is just a static string. MessageExpression must statically evaluate to a string to be accepted.
```

Comment:
I don't think the majority of users will understand what "statically evaluate" means here. Clarify or use a different term?  Is "MessageExpression must evaluate to a string" sufficient?

Reply:
I revised it to read "must evaluate to a string at compilation time" instead, which seems like it'd be pretty readable.